pysilcam/process.py

```diff
@@ -250,9 +252,9 @@ def measure_particles(imbw, imc, settings, timestamp, nnmodel, class_labels):
     '''
     # check the converage of the image of particles is acceptable
     sat_check, saturation = concentration_check(imbw, settings)
-    if sat_check == False:
+    if (sat_check is False):
```

Comment:
```suggestion
    if sat_check is False:
```
As above, although here since we are checking for false, I believe people write `if not sat_check:` (although only in the case sat_check must be a bool).